src/widgets/opnsense/widget.js

```diff
@@ -0,0 +1,18 @@
+
+import genericProxyHandler from "utils/proxy/handlers/generic";
+
+const widget = {
+  api: "{url}/api/{endpoint}",
+  proxyHandler: genericProxyHandler,
+
+  mappings: {
+    activity: {
+      endpoint: "diagnostics/activity/getActivity",
```

Comment:
```suggestion
      endpoint: "diagnostics/activity/getActivity",
      validate: [
        "headers"
      ]
```

As with others, can probably add a validate param